components/transaction-details/transaction-details.module.css

```diff
@@ -7,45 +7,43 @@
 
 .transactionCard {
   display: flex;
-  flex-direction: row;
-  flex-wrap: wrap;
   justify-content: flex-start;
-  margin: 10px 0;
-  width: 500px;
-  background-color: #fdfbfb;
-  border: 1px solid #f4f1f1;
-  border-top-left-radius: 15px;
-  border-bottom-left-radius: 15px;
+  flex-wrap: wrap;
+  width: 100%;
+  border-top: 1px solid #f0e2e7;
+  border-bottom: 1px solid #f0e2e7;;
   cursor: pointer;
+  height:6em;
+  font-size: 17px;
+
 }
 
+
 .transactionCard .action {
   display: flex;
-  flex-direction: row;
   flex-wrap: wrap;
   justify-content: center;
-  width: 110px;
   align-items: center;
-  padding: 15px;
-  border-top-left-radius: 15px;
-  border-bottom-left-radius: 15px;
+  padding:4px;
   color: white;
   font-weight: bold;
-  font-size: 25px;
+  width:4em;
+  height: 4em;
```

Comment:
I might be wrong here, but Doesn't defining height of particular em might lead to overflow of characters?

Reply:
https://css-tricks.com/forums/topic/what-should-be-height-units-in-responsive-design/
Defining in ems rely on nearest parent font-size, so we end with lot of strange numbers in our case but I have not seen the problem you mentioned. Can you please reproduce  this and then we can work on that.